src/app/screens/Onboard/TestConnection/index.tsx

```diff
@@ -33,14 +26,8 @@ export default function TestConnection() {
       setErrorMessage(t("connection_taking_long"));
     }, 45000);
     try {
-      const response = await api.getAccountInfo();
-      const name = response.name;
-      const { alias } = response.info;
-      const { balance: resBalance } = response.balance;
-      const balance =
-        typeof resBalance === "number" ? resBalance : parseInt(resBalance);
-
-      setAccountInfo({ alias, balance, name });
+      await api.getAccountInfo();
+      utils.redirectPage("options.html");
```

Comment:
can we explicitly check that we got valid data from the `getAccountInfo` call and only then redirect? 

this page is about testing the connection so lets also explicitly check this in the code.